CTX.js

```diff
@@ -271,14 +213,30 @@ var mapXMLtoKEV = {
+	return items;
+}
+
+function getZoteroItems() {
+	return zoteroItems;
+}
+
+var exports = {
+	"doImportFromText": doImportFromText,
+	"getItems" : getItems,
+	"getZoteroItems" : getZoteroItems
+}
+
 /** BEGIN TEST CASES **/
 var testCases = [
 	{
 		"type": "import",
-		"input": "<ctx:context-objects xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:schemaLocation='info:ofi/fmt:xml:xsd:ctx http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:ctx' xmlns:ctx='info:ofi/fmt:xml:xsd:ctx'>\n<ctx:context-object timestamp='2010-01-02T16:55:48-05:00' encoding='info:ofi/enc:UTF-8' version='Z39.88-2004' identifier=''>\n <ctx:referent>\n  <ctx:metadata-by-val>\n   <ctx:format>info:ofi/fmt:xml:xsd:journal</ctx:format>\n   <ctx:metadata>\n    <journal xmlns:rft='info:ofi/fmt:xml:xsd:journal' xsi:schemaLocation='info:ofi/fmt:xml:xsd:journal http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:journal'>\n\t <rft:atitle>Acute Myocardial Infarction in the Medicare population: process of care and clinical outcomes</rft:atitle>\n\t <rft:spage>2530</rft:spage>\n\t <rft:date>1992</rft:date>\n\t <rft:stitle>Journal of the American Medical Association</rft:stitle>\n\t <rft:genre>article</rft:genre>\n\t <rft:volume>18</rft:volume>\n\t <rft:epage>2536</rft:epage>\n\t <rft:au>I S Udvarhelyi</rft:au>\n\t <rft:au>C A Gatsonis</rft:au>\n\t <rft:au>A M Epstein</rft:au>\n\t <rft:au>C L Pashos</rft:au>\n\t <rft:au>J P Newhouse</rft:au>\n\t <rft:au>B J McNeil</rft:au>\n\t</journal>\n   </ctx:metadata>\n  </ctx:metadata-by-val>\n </ctx:referent>\n</ctx:context-object>\n</ctx:context-objects>",
+		"input": "<ctx:context-objects xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:schemaLocation='info:ofi/fmt:xml:xsd:ctx http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:ctx' xmlns:ctx='info:ofi/fmt:xml:xsd:ctx'>\\n<ctx:context-object timestamp='2010-01-02T16:55:48-05:00' encoding='info:ofi/enc:UTF-8' version='Z39.88-2004' identifier=''>\\n <ctx:referent>\\n  <ctx:metadata-by-val>\\n   <ctx:format>info:ofi/fmt:xml:xsd:journal</ctx:format>\\n   <ctx:metadata>\\n    <journal xmlns:rft='info:ofi/fmt:xml:xsd:journal' xsi:schemaLocation='info:ofi/fmt:xml:xsd:journal http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:journal'>\\n\\t <rft:atitle>Acute Myocardial Infarction in the Medicare population: process of care and clinical outcomes</rft:atitle>\\n\\t <rft:spage>2530</rft:spage>\\n\\t <rft:date>1992</rft:date>\\n\\t <rft:stitle>Journal of the American Medical Association</rft:stitle>\\n\\t <rft:genre>article</rft:genre>\\n\\t <rft:volume>18</rft:volume>\\n\\t <rft:epage>2536</rft:epage>\\n\\t <rft:au>I S Udvarhelyi</rft:au>\\n\\t <rft:au>C A Gatsonis</rft:au>\\n\\t <rft:au>A M Epstein</rft:au>\\n\\t <rft:au>C L Pashos</rft:au>\\n\\t <rft:au>J P Newhouse</rft:au>\\n\\t <rft:au>B J McNeil</rft:au>\\n\\t</journal>\\n   </ctx:metadata>\\n  </ctx:metadata-by-val>\\n </ctx:referent>\\n</ctx:context-object>\\n</ctx:context-objects>",
```

Comment:
I think it happened when I edit the input of the test and save it again (and/or delete the test temporarily).